### PR TITLE
Feature/cache persistence

### LINKED ISSUE #715: If you restart the app - browser resets cookies
<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

### PR DESCRIPTION
**Pull request by OlegGordiichuk**
- Add cookies store class.
- Apply usage of the cookies store.

Fix #715

### CODE REVIEW

File: Trust/InCoordinator.swift [hunk 1]
```diff
@@ -193,6 +193,7 @@ class InCoordinator: Coordinator {
         navigationController.dismiss(animated: false, completion: nil)
         coordinator.navigationController.dismiss(animated: true, completion: nil)
         coordinator.stop()
+        CookiesStore.delete()
```
Comment (vikmeup) [inline Trust/InCoordinator.swift]: can you add this option when you call clear cache from DApp Browser / Cache

File: Trust/Core/Analytics/AnalyticsEvent.swift [hunk 2]
```diff
@@ -24,6 +24,7 @@ enum AnalyticsEvent {
     case failedToExcludeFromBackup(Error)
     // DAPP browser events
     case failedToLoadTrustPageProvider
+    case failedToLoadCookies
```
Comment (vikmeup) [inline Trust/Core/Analytics/AnalyticsEvent.swift]: No need to track this

File: Trust/InCoordinator.swift (hunk 1 above)
Comment (OlegGordiichuk) [reply Trust/InCoordinator.swift]: Done

File: Trust/Settings/Coordinators/SettingsCoordinator.swift [hunk 3]
```diff
@@ -154,6 +154,7 @@ extension SettingsCoordinator: SettingsViewControllerDelegate {
             delegate?.didPressURL(url, in: self)
         case .clearBrowserCache:
             cleadCache()
+            CookiesStore.delete()
```
Comment (vikmeup) [inline Trust/Settings/Coordinators/SettingsCoordinator.swift]: @OlegGordiichuk can you move this into `clearCache()` function?

**Pull request merged**